### PR TITLE
added new cool feature

### PR DESCRIPTION
**Pull request by zohabylc**

### CODE REVIEW

File: GitHub/GitHub/ViewController.swift [hunk 1]
```diff
@@ -14,6 +14,10 @@ class ViewController: UIViewController {
     super.viewDidLoad()
     // Do any additional setup after loading the view.
   }
+  
+  func newFunction() {
```
Comment (Zoha131) [inline GitHub/GitHub/ViewController.swift:18]: Why have you added this function?

Comment (zohabylc) [reply GitHub/GitHub/ViewController.swift:18]: I have changed as per your suggestions

Comment (zohabylc) [reply GitHub/GitHub/ViewController.swift:18]: changed what i have asked to do

File: GitHub/GitHub/ViewController.swift [hunk 2]
```diff
@@ -14,6 +14,11 @@ class ViewController: UIViewController {
...
+    print("Hello Github")
+    print("this is new change after change request")
```
Comment (Zoha131) [inline GitHub/GitHub/ViewController.swift:20]: What is the purpose of this line?

**Pull request merged**